scripts/cleanup_openstack_instance_snapshot.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
```

Comment:
we may want to get it on py3 since we are moving everything else to py3.

Reply:
right now, our all other scripts are based on python2, i would suggest , we can migrate all at once when we are ready for py3.

Reply:
I think I will still vote on making it py3 now than to migrate later. Let's see what the other reviewers have to say.

Reply:
Not correct that all the scripts are py2. Cleanup scripts are explicitly py3 because of their use of starmap.